positron/src/positron/fastapi_template/app.py.tmpl

```diff
@@ -0,0 +1,100 @@
+from __future__ import annotations
+
+import asyncio
+import socket
+from threading import Thread
+
+import toga
+from asgiref.compatibility import guarantee_single_callable
+
+from .server import app as fastapi_app
```

Comment:
Putting this import at the top level means the import is performed *before* the app starts; it would be preferable to defer this until the app is running.